### DISCUSSION TITLE
When is Version 1.0 coming out?

### DISCUSSION THREAD

**Discussion by LuckyGold7** [CodeAsherPlayz/Full-Login-Page#21] [Q&A] [answered]
I would like to know when Version 1.0 is coming out as it is been awhile since Version 1.0-Beta came out and I would like to get the latest Version 1.0. Thanks!

**Comment by CodeAsherPlayz** [comment, marked as answer]
Hi @LuckyGold7!

As mentioned earlier in #13 and #22, as well as the milestones, Version 1.0 will be coming out shortly. This is because we are still working on Version 2.0-Beta which will be one of the biggest updates for the software. After finalising Version 2.0-Beta, we will be looking into Version 1.0-Beta Issue #12, which mentions about the log-in page being squashed up. After finalising both, we will roll out Version 1.0 first, and Version 2.0-Beta by the end of January 2021. Sorry for keeping you waiting.

~ CodeAsherPlayz

**Reply by LuckyGold7** [reply]
Thank you so much!
